assets/js/mixins/formatter.test.js

```diff
@@ -51,33 +58,40 @@ describe("fmtWh", () => {
   test("should format with units", () => {
     expect(fmt.fmtWh(0, POWER_UNIT.AUTO)).eq("0 kWh");
     expect(fmt.fmtWh(1200000, POWER_UNIT.AUTO)).eq("1,2 MWh");
-    expect(fmt.fmtWh(0, POWER_UNIT.MW)).eq("0,0 kWh");
+    expect(fmt.fmtWh(0, POWER_UNIT.MW)).eq("0 kWh");
     expect(fmt.fmtWh(1200000, POWER_UNIT.MW)).eq("1,2 MWh");
-    expect(fmt.fmtWh(0, POWER_UNIT.KW)).eq("0,0 kWh");
+    expect(fmt.fmtWh(0, POWER_UNIT.KW)).eq("0 kWh");
     expect(fmt.fmtWh(1200000, POWER_UNIT.KW)).eq("1.200,0 kWh");
     expect(fmt.fmtWh(0, POWER_UNIT.W)).eq("0 kWh");
     expect(fmt.fmtWh(1200000, POWER_UNIT.W)).eq("1.200.000 Wh");
   });
   test("should format without units", () => {
     expect(fmt.fmtW(0, POWER_UNIT.AUTO, false)).eq("0");
     expect(fmt.fmtW(1200000, POWER_UNIT.AUTO, false)).eq("1,2");
-    expect(fmt.fmtWh(0, POWER_UNIT.MW, false)).eq("0,0");
+    expect(fmt.fmtWh(0, POWER_UNIT.MW, false)).eq("0");
     expect(fmt.fmtWh(1200000, POWER_UNIT.MW, false)).eq("1,2");
-    expect(fmt.fmtWh(0, POWER_UNIT.KW, false)).eq("0,0");
+    expect(fmt.fmtWh(0, POWER_UNIT.KW, false)).eq("0");
     expect(fmt.fmtWh(1200000, POWER_UNIT.KW, false)).eq("1.200,0");
     expect(fmt.fmtWh(0, POWER_UNIT.W, false)).eq("0");
     expect(fmt.fmtWh(1200000, POWER_UNIT.W, false)).eq("1.200.000");
   });
   test("should format a given number of digits", () => {
+    expect(fmt.fmtWh(0, POWER_UNIT.AUTO, true, 3)).eq("0,0 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.AUTO, true, 0)).eq("12 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.AUTO, true, 1)).eq("12,3 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.AUTO, true, 2)).eq("12,35 kWh");
+
+    expect(fmt.fmtWh(0, POWER_UNIT.MW, true, 3)).eq("0,0 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.MW, true, 0)).eq("0 MWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.MW, true, 1)).eq("0,0 MWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.MW, true, 2)).eq("0,01 MWh");
+
+    expect(fmt.fmtWh(0, POWER_UNIT.KW, true, 3)).eq("0,0 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.KW, true, 0)).eq("12 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.KW, true, 1)).eq("12,3 kWh");
     expect(fmt.fmtWh(12345, POWER_UNIT.KW, true, 2)).eq("12,35 kWh");
+
+    expect(fmt.fmtWh(0, POWER_UNIT.W, true, 3)).eq("0,0 kWh");
```

Comment:
I guess we had a misunderstanding here. I'd only use the 0,0 kWh in AUTO format. When a specific unit is requested the function should always honor that.

Also if digits param is provided I'd always honor that. I guess in practice digits and auto format are not used together.

Reply:
Ok, no problem.